_extensions/auto-register/change-login-link.html

```diff
@@ -12,19 +12,21 @@
  * Description: Creates a custom email template for EDD emails.
  * Author: Easy Digital Downloads
  * Author URI: https://easydigitaldownloads.com/
- * Version: 1.0
+ * Version: 1.0.1
  */
 
 // replace the default WordPress login link in Auto Register email
-function custom_edd_auto_register_login_link( $login_url ) {
+function custom_edd_auto_register_login_link( $default_email_body ) {
 
-	// replace http://yoursite.com with your real domain where WordPress is installed
 	// replace http://yoursite.com/sign-in with the URL you want to send users to
-	$login_url = str_replace(
-		'http://yoursite.com/wp-login.php',
+	$login = wp_login_url();
+
+	$default_email_body = str_replace(
+		$login,
```

Comment:
You could just change this to `wp_login_url()` directly and remove the `$login` variable. Since it's only used once.

Reply:
Adjusted. Was not sure if using a function inside the str_replace will work. Should be good now